server/server-config.js

```diff
@@ -11,6 +11,6 @@ app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname + '/../client')));
 
 app.get('/', handler.renderIndex);
```

Comment:
If we're using express.static, we shouldn't need to manually handle the '/' endpoint for serving static pages.